circuits/cpp/src/aztec3/circuits/abis/c_bind.test.cpp

```diff
@@ -191,7 +191,10 @@ TEST(abi_tests, compute_function_leaf)
 {
     // Construct FunctionLeafPreimage with some randomized fields
     auto const preimage = FunctionLeafPreimage<NT>{
-        .function_selector = engine.get_random_uint32(),
+        .function_selector =
+            FunctionSelector<NT>{
```

Comment:
nit: I would think this could leave off the type and just have = { ...}

Reply:
Thanks Adam. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/1800 .